packages/api/src/routers/api/chart.ts

```diff
@@ -48,37 +48,59 @@ router.get('/services', async (req, res, next) => {
     const MAX_NUM_GROUPS = 2000;
 
     const simpleCache = new SimpleCache<
-      Awaited<ReturnType<typeof clickhouse.getMultiSeriesChart>>
+      Awaited<ReturnType<typeof clickhouse.getMultiSeriesChart>>[]
     >(`chart-services-${teamId}`, ms('10m'), () =>
-      clickhouse.getMultiSeriesChart({
-        series: [
-          {
-            aggFn: clickhouse.AggFn.Count,
-            groupBy: targetGroupByFields,
-            table: 'logs',
-            type: 'table',
-            where: '',
-          },
-        ],
-        endTime,
-        granularity: undefined,
-        maxNumGroups: MAX_NUM_GROUPS,
-        startTime,
-        tableVersion: team.logStreamTableVersion,
-        teamId: teamId.toString(),
-        seriesReturnType: clickhouse.SeriesReturnType.Column,
-      }),
+      Promise.all([
+        clickhouse.getMultiSeriesChart({
+          series: [
+            {
+              aggFn: clickhouse.AggFn.Count,
+              groupBy: targetGroupByFields,
+              table: 'logs',
+              type: 'table',
+              where: '',
+            },
+          ],
+          endTime,
+          granularity: undefined,
+          maxNumGroups: MAX_NUM_GROUPS,
+          startTime,
+          tableVersion: team.logStreamTableVersion,
+          teamId: teamId.toString(),
+          seriesReturnType: clickhouse.SeriesReturnType.Column,
+        }),
+        clickhouse.getMultiSeriesChart({
```

Comment:
The quick hack is to fetch service groups separately and join with custom attributes later